config/initializers/filter_parameter_logging.rb

```diff
@@ -0,0 +1,4 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
I don't think this file is needed at all, there aren't any passwords used in this repo.